index.php

```diff
@@ -0,0 +1,30 @@
+<?php include('connect.php'); ?><!DOCTYPE html>
+<html>
+  <head>
+    <title>Make Your Pizza</title>
+  </head>
+  <body>
+    <div class="container">
+      <h1>Choose Your Toppings</h1>
+      <form action="review.php" method="POST">
+        <div class="group">
+          <label for="cheese">Cheese</label><?php while($row = mysqli_fetch_assoc($results['cheeses'])) { ?>
+          <input name="cheese" type="radio" value="&lt;?php echo $row['name']; ?&gt;"><span><?php echo $row['name'] ?>: $<?php echo $row['price']; ?></span><?php } ?>
+        </div>
```

Comment:
I can't run this right now to test.  Let me know if it does look that way.  I suspect your review.php file shows:
- &lt;?php echo $row['name']; ?&gt;
- &lt;?php echo $row['name']; ?&gt;
- &lt;?php echo $row['name']; ?&gt;
- &lt;?php echo $row['name']; ?&gt;
- &lt;?php echo $row['name']; ?&gt;

If that's not the case, then maybe I missed something in how this works.